badge_server/main.py

```diff
@@ -76,6 +77,23 @@ class BadgeStatus(enum.Enum):
     OUTDATED_DEPENDENCY = 'OUTDATED_DEPENDENCY'
     SUCCESS = 'SUCCESS'
 
+    @classmethod
+    def get_highest_status(cls, statuses: Iterable[enum.Enum]):
+        """Gets the highest BadgeStatus.
+
+        This method works because `cls.__members__` is an ordered dict.
+
+        Args:
+            statuses: A list of BadgeStatuses.
+
+        Returns:
+            A BadgeStatus. An error is thrown if no BadgeStatus exists.
```

Comment:
http://google.github.io/styleguide/pyguide.html#244-decision

```suggestion
            The BadgeStatus found in `statuses` that has the highest priority. For example, INTERNAL_ERROR would be returned if `statuses` contained INTERNAL_ERROR and OUTDATED_DEPENDENCY.
            
  Raises:
     ValueError: if `statuses` does not contain a valid BadgeStatus.
```

Reply:
done